openedx/core/djangoapps/self_paced/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utilities for the self_paced app.
+"""
+from lms.djangoapps.certificates.api import set_cert_generation_enabled
```

Comment:
I think it's incorrect to import things from 'lms' into 'openedx' -- the 'lms' area is for code specific to the LMS IDA, whereas 'openedx' is for code generic to more than one IDA (in this case Studio and LMS).  So it looks like there is a decision that needs to be made here, in terms of where 'set_cert_generation_enabled' and 'toggle_cert_generation' should live.